capstone-ui/src/index.js

```diff
@@ -1,12 +1,13 @@
 import React from "react";
 import ReactDOM  from "react-dom/client";
+import { BrowserRouter } from "react-router-dom";
 import App from "./components/App/App";
 import "./index.css"
 
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <BrowserRouter>
```

Comment:
👍  Good change, we need BrowserRouter here.
Do we not want React.StrictMode as well?

Reply:
I put back the strict Mode. I removed when I was fixing my routes.